tools/tracy/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.13...3.18 FATAL_ERROR)
+project(inviwo-tracy DESCRIPTION "Inviwo tracy wrapper" LANGUAGES CXX)
+
+set(HEADER_FILES
+    include/inviwo/tracy/tracy.h
+    include/inviwo/tracy/tracyopengl.h
+)
+
+add_library(inviwo-tracy INTERFACE)
+add_library(inviwo::tracy ALIAS inviwo-tracy)
+
+target_include_directories(inviwo-tracy INTERFACE include)
+
+option(IVW_USE_TRACY "Enable profiling using Tracy" OFF)
+if(IVW_USE_TRACY)
+	find_package(unofficial-tracy REQUIRED)
```

Comment:
Link to this one? 
Way not official one, elaborate...

Reply:
it does not have an official vcpkg port. We have our own in inviwo

Reply:
Ok, I read in the docs that they use vcpkg, but it looks like that was only for the server side.

Reply:
Yes, I have created a vcpkg port, but I have not pushed that yet, have yet to decide where is should go...